test/chrome/chromeUtils.test.ts

```diff
@@ -318,6 +318,27 @@ suite('ChromeUtils', () => {
                 chromeUtils.getMatchingTargets(targets, 'http://localhost'),
                 targets);
         });
+
+        test('handles double dots', () => {
+            const targets = makeTargets('http://localhost/app', 'http://localhost/site/../folder/../app');
```

Comment:
Thanks for writing tests, but the target won't have `..`, and these tests aren't actually testing that the `..` is resolved, right?